tools/python/influx-migration/influx_migration.py

```diff
@@ -280,7 +280,7 @@ def health_check(host, token, skip_verify):
         client = InfluxDBClient(url=host,
             token=token, timeout=MILLISECOND_TIMEOUT, verify_ssl=not skip_verify)
         if not client.ping():
-            raise InfluxDBError(message=f"{host} ping failed")
+            raise InfluxDBError(message=f"InfluxDB API call to {host}/ping failed")
```

Comment:
If more clarification is needed to let users know whether their hosts are formatted correctly, let me know.

Reply:
Can we log original error?

Reply:
`client.ping()` just [returns a bool](https://influxdb-client.readthedocs.io/en/stable/api.html#influxdb_client.InfluxDBClient.ping). An exception is logged by `influxdb_client`:
```python
def ping(self) -> bool:
"""
Return the status of InfluxDB instance.

:return: The status of InfluxDB.
"""
ping_service = PingService(self.api_client)

try:
    ping_service.get_ping()
    return True
except Exception as ex:
    logger.debug("Unexpected error during /ping: %s", ex)
    return False
```
but I haven't been able to get this log to output, even with `--log-level debug`. I'll see what I can do.

Reply:
Debug logging is enabled when the client has `debug=True`. With my latest change, if a user uses `--log-level debug` the client used for the health check will output debug logs.

The format is not consistent with how we format our logs. This is an example output:
```shell
>>> Request: 'GET 1.2.2.1.3/ping?'
DEBUG: rest.py: >>> Request: 'GET 1.2.2.1.3/ping?'
>>> Authorization: ***
DEBUG: rest.py: >>> Authorization: ***
>>> User-Agent: influxdb-client-python/1.41.0
DEBUG: rest.py: >>> User-Agent: influxdb-client-python/1.41.0
>>> Content-Type: application/json
DEBUG: rest.py: >>> Content-Type: application/json
>>> Body: None
DEBUG: rest.py: >>> Body: None
DEBUG: retry.py: Converted retries value: False -> Retry(total=False, connect=None, read=None, redirect=0, status=None)
DEBUG: connectionpool.py: Starting new HTTP connection (1): 1.2.2.1.3:80
DEBUG: influxdb_client.py: Unexpected error during /ping: <urllib3.connection.HTTPConnection object at 0x10422a7e0>: Failed to resolve '1.2.2.1.3' ([Errno 8] nodename nor servname provided, or not known)
ERROR: influx_migration.py: InfluxDB API call to 1.2.2.1.3/ping failed
ERROR: influx_migration.py: Health check for source host failed
ERROR: influx_migration.py: Migration failed. Exiting . . .
```

Reply:
This may be a good spot to add additional guidance on what may have caused this error and how users can fix it. We could note that the host may be unreachable or the url specified may not be complete. Ensure that the url has the appropriate scheme, address, and port values...

Reply:
I have added more logs to help users:
```shell
ERROR: influx_migration.py: Host is missing scheme
ERROR: influx_migration.py: Host is missing port
ERROR: influx_migration.py: The expected format for host urls is <scheme>:<domain>:<port>. For example, http://127.0.0.1:8086
ERROR: influx_migration.py: InfluxDB API call to 1.2.2.1.3/ping failed
ERROR: influx_migration.py: Health check for source host failed
ERROR: influx_migration.py: Migration failed. Exiting . . .
```